docs/build/identity.rs/1.3/docs/how-tos/verifiable-credentials/zero-knowledge-selective-disclosure.mdx

```diff
@@ -0,0 +1,151 @@
+---
+sidebar_label: Zero Knowledge Selective Disclosure
+description: Zero Knowledge selectively disclosable VCs.
+image: /img/Identity_icon.png
+tags:
+  - verifiable
+  - credentials
+  - Zero Knowledge
+  - Disclosure
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Zero Knowledge Selective Disclosure (ZK-SD-VCs)
+ZK-SD-VCs allow holders to verify their VCs without having to disclose to verifiers the entire VC's claim set. 
+This is done through the creation of a Zero Knowledge Proof (ZKP) that guaranties the integrity and authenticity
+of the VC, even when only partially disclosed to the verifier.
+
+:::note
+Although ZK-SD-VCs offer similar functionalities to [SD-JWT VCs](../selective-disclosure) - at least on a high level - they rely on completely different
+concepts and security concerns. For an user, the most notable difference is the shifted capability of choosing which fields can
+be concealed from a verifier. For ZK-SD-VCs it's the holder that has total control over which parts of the credential can be 
+undisclosed, whereas for SD-JWT VCs it's the issuer that decides which fields may be concealed by the holder.
+:::
+
+## Concepts
+### Issuance
+The issuer of a ZK-SD-VC creates the credential, signs it using the [BBS+](https://identity.foundation/bbs-signature/draft-irtf-cfrg-bbs-signatures.html) signature scheme
+and sends both the credential and the signature to the holder. To facilitate this process, the credential is first encoded 
+as a [JSON Proof Token](https://www.ietf.org/archive/id/draft-ietf-jose-json-proof-token-02.html) (JPT), which is then used as the payload of a 
+[JSON Web Proof](https://www.ietf.org/archive/id/draft-jmiller-jose-json-web-proof-00.html) (JWP) and sent to the holder as JPT. 
```

Comment:
This is the correct link: https://www.ietf.org/archive/id/draft-ietf-jose-json-web-proof-02.html